src/test/resources/application.yaml

```diff
@@ -0,0 +1,50 @@
+quarkus:
```

Comment:
@ruromero you should also add the property to generate the jar as mutable-jar out-of-box (quarkus.package.type=mutable-jar), as this is a pre-condition for users to change the default database.

Reply:
I have added it although this is part of Quarkus 2.2.x which is not yet released, meaning that for now the property will be ignored